site/gatsby-site/playwright/e2e/dynamicCite.spec.ts

```diff
@@ -0,0 +1,144 @@
+import { test } from '../utils';
+import incident10 from '../fixtures/incidents/incident10.json';
+import { conditionalIntercept, waitForRequest } from '../utils';
+import config from '../config';
+import { expect } from '@playwright/test';
+
+test.describe('Dynamic Cite pages', () => {
+  const incidentId = 10;
+  const url = `/cite/${incidentId}`;
+
+  test('Successfully loads', async ({ page }) => {
+    await page.goto(url);
+  });
+
+  test("Shouldn't display live data option if it's not logged in", async ({ page }) => {
+    await page.goto(url);
+    await expect(page.locator('[data-cy="toogle-live-data"]')).not.toBeVisible();
+  });
+
+  test('Should load dynamic Incident data', async ({ page, login }) => {
```

Comment:
The original test has this condition `!Cypress.env('isEmptyEnvironment') && Cypress.env('e2eUsername') && Cypress.env('e2ePassword')` so I think the new one should have the `skipOnEmptyEnvironment`